templates/docs/patterns/grid/index.md

```diff
@@ -212,15 +92,15 @@ To import just this component into your project, copy the snippet below and incl
 @import 'vanilla-framework';
 @include vf-base;
 
-@include vf-p-grid;
+@include vf-p-grid-8;
 ```
 
 For more information see [Customising Vanilla](/docs/customising-vanilla/) in your projects, which includes overrides and importing instructions.
 
 ## React
 
-You can use grid in React by installing our react-component library and importing `Row` and `Col` components.
+You can use the grid in React by installing our react-component library and importing `GridRow` and `GridCol` components.
```

Comment:
We should not link to something that doesn't exist yet. I think it's fine to release it first without React support and add to React later (once we decide if it should be a new component or prop on existing one).

Reply:
Fixed